### DISCUSSION TITLE
Where to find the information what string pattern a setter value for a property need?

### DISCUSSION THREAD

**Discussion by Agredo** [xamarin/Xamarin.Forms#14415] [Q&A] [answered]
I want to now where I can get the information what the right string is for a value inside a Setter

I Cant find any page where I can read how to write the value string for a ItemsLayout.
![image](https://user-images.githubusercontent.com/16531090/125456840-7eea5f7f-1279-41d3-b53c-d537e7722aed.png)

Now I need to set the VericalSpacing and HrozontalSpacing for a VerticalGrid but I dont know how. 

Thank you :)

**Comment by sjordanGSS** [comment, marked as answer]
This is quite an old thread now, but have you tried something like:
```
<Setter Property="CollectionView.ItemsLayout">
  <Setter.Value>
    <GridItemsLayout Orientation="Vertical" Span="3" VerticalItemSpacing="5" HorizontalItemSpacing="5" />
  </Setter.Value>
</Setter>
```

**Reply by Agredo** [reply]
Haha this is a so simple solution thank you! :D
